Doc/whatsnew/3.11.rst

```diff
@@ -176,14 +176,33 @@ The :option:`-X` ``no_debug_ranges`` option and the environment variable
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+Exception Groups and ``except*`` (PEP 654)
```

Comment:
Could we be consistent about PEP numbers in Feature section titles? Right now the type hint PEP sections are in the form `PEP NNN: <Title>`, this section and the one below is in the form `<Title> (PEP NNN)`, and the two above it don't mention the PEP number at all in the title.

IMO, it would seem to make sense to either elide the PEP numbers in the New Features section titles (as I do for the section below in #95914 , since they don't add much value for most users and are already mentioned/linked in the section body and Summary), or consistently use them with the same format between these two sections (I can take care of the others in my separate PRs, but it would be nice to be consistent here).